app/src/main/java/org/fossasia/badgemagic/ui/DrawerActivity.kt

```diff
@@ -62,13 +62,22 @@ class DrawerActivity : BaseActivity(), NavigationView.OnNavigationItemSelectedLi
         if (intent.action == Intent.ACTION_VIEW)
             importFile(intent)
 
+        defaultFragment()
+
         setupDrawerAndToolbar()
 
         prepareForScan()
 
         handleIfReceiveIntent()
     }
 
+    private fun defaultFragment() {
+        if (supportFragmentManager.findFragmentById(R.id.frag_container) == null) {
```

Comment:
Why is this needed?

Reply:
@iamareebjamal initially there is no fragment so I have just set a default fragment if no fragment is there.

Reply:
Change method name to setDefaultFragment